src/app/send/add-edit.component.html

```diff
@@ -1,106 +1,159 @@
+                            <div class="subtext">{{'sendTextDesc' | i18n}}</div>
                         </div>
                         <div class="box-content-row box-content-row-checkbox" appBoxRow *ngIf="send.type === sendType.Text">
                             <label for="hideText">{{'textHiddenByDefault' | i18n}}</label>
                             <input id="hideText" name="hideText" type="checkbox" [(ngModel)]="send.text.hidden">
                         </div>
-                    </div>
+                    </ng-container>
+                </div>
+            </div>
+            <div class="box">
+                <div class="box-header">
+                    {{'options' | i18n}}
                 </div>
-                <div class="box">
-                    <div class="box-header">
-                        {{'options' | i18n}}
+                <div class="box-content">
+                    <div class="box-content-row" appBoxRow>
+                        <label for="deletionDate">{{'deletionDate' | i18n}}</label>
+                        <input id="deletionDate" type="datetime-local" name="deletionDate"
+                            [(ngModel)]="deletionDate" required placeholder="MM/DD/YYYY HH:MM AM/PM" *ngIf="editMode">
+                        <select id="deletionDate" name="DeletionDateSelect" [(ngModel)]="deletionDateSelect" required *ngIf="!editMode">
+                            <option *ngFor="let o of deletionDateOptions" [ngValue]="o.value">{{o.name}}
+                            </option>
+                        </select>
+                        <div class="subtext">{{'deletionDateDesc' | i18n}}</div>
                     </div>
-                    <div class="box-content">
-                        <div class="box-content-row" appBoxRow *ngIf="editMode">
-                            <label for="deletionDate">{{'deletionDate' | i18n}}</label>
-                            <input id="deletionDate" type="datetime-local" name="deletionDate"
-                                [(ngModel)]="deletionDate" required placeholder="MM/DD/YYYY HH:MM AM/PM">
-                            <div class="subtext">{{'deletionDateDesc' | i18n}}</div>
-                        </div>
-                        <div class="box-content-row" appBoxRow>
-                            <label for="expirationDate">{{'expirationDate' | i18n}}</label>
-                            <input id="expirationDate" type="datetime-local" name="expirationDate" [(ngModel)]="expirationDate">
-                            <div class="subtext">{{'expirationDateDesc' | i18n}}</div>
-                        </div>
+                    <div class="box-content-row" *ngIf="deletionDateSelect === 0 && !editMode">
+                        <input id="deletionDateCustom" type="datetime-local" name="deletionDate" 
+                            [(ngModel)]="deletionDate" required placeholder="MM/DD/YYYY HH:MM AM/PM" [readOnly]="disableSend">
```

Comment:
This readonly tag doesn't appear to do anything in my attempts

Reply:
Addressed above